cli/spark/endpoints/init.py

```diff
@@ -8,14 +8,29 @@
 def setup_parser(parser: argparse.ArgumentParser):
     parser.add_argument('--global', dest='global_flag', action='store_true',
                         help="Create a .aztk/ folder in your home directory for global configurations.")
+    software_parser = parser.add_mutually_exclusive_group()
+    software_parser.add_argument('--python', action="store_true", required=False)
+    software_parser.add_argument('--r', '--R', action="store_true", required=False)
+    software_parser.add_argument('--java', action="store_true", required=False)
+    software_parser.add_argument('--scala', action="store_true", required=False)
+
 
 def execute(args: typing.NamedTuple):
+    # software_specific init
+    if args.python:
+        docker_repo = constants.DEFAULT_SPARK_PYTHON_DOCKER_REPO
+    elif args.r:
```

Comment:
does this need to change to args.r || args.R?

Reply:
Both `--r` and `--R` go to `args.r`